dev/bots/docs.sh

```diff
@@ -20,7 +20,7 @@ function generate_docs() {
     # Install and activate dartdoc.
     # NOTE: When updating to a new dartdoc version, please also update
     # `dartdoc_options.yaml` to include newly introduced error and warning types.
```

Comment:
Is there supposed to be a dartdoc_options.yaml update too?

Reply:
No new error or warning types have been introduced in this dartdoc release [1], so no update to that file is necessary! Thanks for watching out for this, though!

[1] https://pub.dev/packages/dartdoc/changelog#600